org.scala-ide.refactoring/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.scala-ide</groupId>
+    <artifactId>org.scala-ide.p2-toolchain</artifactId>
+    <version>4.4.0-SNAPSHOT</version>
+    <relativePath>../org.scala-ide.p2-toolchain/pom.xml</relativePath>
+  </parent>
+
+  <artifactId>org.scala-ide.refactoring</artifactId>
+  <description>The local p2 repo for scala-refactoring</description>
+  <packaging>pom</packaging>
+  <version>4.4.0-SNAPSHOT</version>
```

Comment:
This version is the one of the P2 repo, it is only used internally in our build process. I don't think users will ever see it. Keeping this version in sync with the version of the IDE for simplicity reasons is probably the best.